### DISCUSSION TITLE
해외주식 compatibility

### DISCUSSION THREAD

**Discussion by softmarshmallow** [elbakramer/koapy#9] [Q&A]
해외주식도 문제 없이 거래 가능한가요? 이베스트 에서 넘어와서 키움 api 는 아직 뜯어보지 못한 상태입니다. 키움에서는 koa 가 둘다 지원한다고는 하네요

**Comment by elbakramer** [comment]
해외쪽은 또 API 가 따로 있는걸로 압니다.
https://fx.kiwoom.com/fxk.template.do?m=m0310020000&

다만 이거는 제가 검토해본적이 없어서 뭐라 말씀드릴게 없네요.
아마도 기본적인 구성은 비슷하긴 할건데, 메소드명이나 이벤트 콜백등을 조금씩 바꿔줘야 하지 않을까 싶습니다.

**Reply by softmarshmallow** [reply]
혹시 지원 예정이 있으신지... 여쩌봅니다 !

**Reply by elbakramer** [reply]
언젠가는 해외쪽도 지원 추가하는 것도 나쁘지 않겠다 정도의 생각이지만요..
사실상 지원 예정이 없다고 보시는게 맞을 것 같습니다.
긍정적인 답변을 드리지 못해서 죄송합니다.

**Reply by elbakramer** [reply]
잠깐 검토해봤는데 해외쪽은 OPEN API 사용하는데 이용료가 필요한 이슈가 있어서 개발이 더욱 어려울 것 같습니다.
